src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -24,8 +24,6 @@
     <Compile Remove="commands\utility\FormatAndOutput\OutGridView\OutWindowProxy.cs" />
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.159.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 91
91:             $result.Replies[0].Address              | Should -BeExactly $realAddress

```
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.159.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 106
106:             $result1.Replies[0].Address              | Should -BeExactly $realAddress

```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.159.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 238
238:             $pingReplies = $replies[-1].PingReplies

```